explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -148,10 +156,25 @@ object AladinCell extends ModelOptics with AladinCommon:
           .void
       case _       => Callback.empty
     }
-    (offsetView, offsetChangeInAladin, offsetOnCenter)
+    (offsetChangeInAladin, offsetOnCenter)
   }
 
-  private val component =
+  def setVariable(root: Option[HTMLElement], variableName: String, value: Int): Callback =
+    root
+      .map(root =>
+        Callback(
+          root.style.setProperty(s"--aladin-image-$variableName", s"${value / 100.0}")
+        )
+      )
+      .getOrEmpty
+
+  private val unsafeRangeLens: Lens[TargetVisualOptions.ImageFilterRange, Double] =
+    Lens[TargetVisualOptions.ImageFilterRange, Double](_.value.toDouble)(x =>
+      _ =>
+        refineV[TargetVisualOptions.FilterRange](x.toInt).toOption
+          .getOrElse(sys.error("Not allowed"))
```

Comment:
Would it complicate things too much if this was an `Optional` instead of a `Lens`? (Or a `Lens[.., Option[Double]]`)